config/default/kustomization.yaml

```diff
@@ -6,16 +6,11 @@ resources:
 - ../rbac
 - ../manager
 - ../webhook
+- ../prometheus
```

Comment:
So since the _prometheus_ directory is already included do you need to `add` it with kustomize as well?

Reply:
Not sure I understood this one. It's already in kustomize. You can deploy using `make deploy` or with a bundle. Both way will create `ServieMonitor` and all the necessary things.

Reply:
My point is was that if the `ServiceMonitor` was already part of the kustomize do you need to `add` it again for the `make deploy` target.